### PR TITLE
Enclave env parsing refactor

### PR DESCRIPTION
**Pull request by Cashmaney**
Made the code inside the enclave a little more modular and readable. Also retouching certain areas of the code as I go. Hopefully nothing too dramatic breaks, but we'll see

### CODE REVIEW

File: cosmwasm/enclaves/shared/contract-engine/src/contract_operations.rs [hunk 1]
```diff
@@ -56,40 +55,42 @@ pub fn init(
...
-            env_v010.contract.address, err
+            "init got an error while trying to deserialize env input bytes into json {:?}: {}",
+            String::from_utf8_lossy(&env),
+            err
         );
         EnclaveError::FailedToDeserialize
     })?;
 
-    trace!("init env_v010: {:?}", env_v010);
+    let (sender, contract_address, block_height, sent_funds) = base_env.get_verification_params();
```
Comment (liorbond) [inline cosmwasm/enclaves/shared/contract-engine/src/contract_operations.rs:69]: why don't you print the params anymore?

File: cosmwasm/enclaves/shared/contract-engine/src/contract_operations.rs [hunk 2]
```diff
@@ -56,40 +55,42 @@ pub fn init(
...
 
-    let canonical_sender_address = CanonicalAddr::from_human(&env_v010.message.sender).map_err(|err| {
+    let canonical_contract_address = CanonicalAddr::from_human(&contract_address).map_err(|err| {
+        warn!(
+            "init got an error while trying to deserialize env_v010.contract.address from bech32 string to bytes {:?}: {}",
```
Comment (liorbond) [inline cosmwasm/enclaves/shared/contract-engine/src/contract_operations.rs:73]: it's not enc_v010 anymore :)

**Review by liorbond — APPROVED** [review]

**Comment by liorbond** [comment]
@Cashmaney can you close this PR?

**Comment by Cashmaney** [comment]
Closed (merged into another branch already)

**Pull request closed without merging**